rfcs/041-item-statuses/README.md

```diff
@@ -0,0 +1,311 @@
+# RFC 041: Statuses in the items API
+
+As part of the [requesting API](../039-requesting-api-design), we need to present an item status.
+We already have a model that we'll use:
+
+```json
+"status": {
+  "id" : "available",
+  "label" : "Available",
+  "type": "ItemStatus"
+}
+```
+
+but we need to decide what value it can take.
+
+This RFC proposes a set of values that we will present through this API.
+
+
+
+## Guiding principles
+
+1.  **We should not assume requests are sent to a particular service.**
+
+    Although the initial implementation will be making requests for items in Sierra, we may add support for requesting from other systems in a future update.
+    We should not assume the use of Sierra, or of Sierra item statuses.
+
+2.  **A user's list of holds is private.**
+
+    Alice should not be able to find out what items Bob has placed on hold, and vice versa.
+
+3.  **The status information will be publicly visible, and displayed on the works page.**
+
+    For consistency, we should use the same terminology as will be used on the works page.
+    We should assume users will see these statuses, and not expose information about the library workings.
+
+
+
+## Existing Sierra statuses
+
+This section describes the existing status data in Sierra items.
+
+Sierra presents the item status in the [Item STATUS field](https://documentation.iii.com/sierrahelp/Default.htm#sril/sril_records_fixed_field_types_item.html#STATUS).
+There are standard codes supplied as part of Sierra, but libraries can define additional item status values.
+
+This is a list of item statuses in use at Wellcome, based on the reporting cluster data.
+(Deleted and suppressed items have been omitted from this analysis.)
+
+<table>
+  <tr>
+    <th>code</th>
+    <th>name</th>
+    <th>count</th>
+    <th>comment</th>
+  </tr>
+  <tr>
+    <td><code>-</code></td>
+    <td>Available</td>
+    <td>616,145</td>
+    <td>
+      The Sierra docs say this status is "items that can circulate", and that "the item retains this status even when checked out".
+      We should test with this use case, and make sure the items API responds appropriately.
```

Comment:
Finding out what actually happens to the item status in Sierra when a hold is placed and when the item is available for pickup is crucial here. I know we can't test right now, but we need to find out the answer to that question in order to work out what the utility of item status even is.

Reply:
BLARGH.

I have a hold on item 1277629. We can see the hold count, but the status stays the same:

```
 'copyNo': 2,
 'holdCount': 1,
 'status': {'code': '-', 'display': 'Available'},
```

<img width="788" alt="Screenshot 2021-05-07 at 15 05 21" src="https://user-images.githubusercontent.com/301220/117461581-a6255200-af45-11eb-97a4-03e1e838948d.png">


<img width="1071" alt="Screenshot 2021-05-07 at 15 04 42" src="https://user-images.githubusercontent.com/301220/117461521-90b02800-af45-11eb-9b08-ed1b48347b57.png">